pkg/kubectl/cmd/cmd.go

```diff
@@ -65,6 +65,7 @@ Find more information at https://github.com/GoogleCloudPlatform/kubernetes.`,
 	cmds.AddCommand(NewCmdClusterInfo(f, out))
 	cmds.AddCommand(NewCmdApiVersions(f, out))
 	cmds.AddCommand(NewCmdVersion(f, out))
+	cmds.AddCommand(NewCmdValidateCluster(f, out))
```

Comment:
I take it back. If this is just component status, we should call it `component-status`.

If we versioned the APIs, it could be:
```
kubectl get cs
```